cybergis_compute_client/UI.py

```diff
@@ -3,6 +3,7 @@
 import ipywidgets as widgets
 from ipyfilechooser import FileChooser
 from IPython.display import Markdown, display, clear_output
+from .MarkdownTable import *  # noqa
```

Comment:
We can ignore it because we are in a rush, but in the future try to minimize star (\*) imports.

Reply:
I removed all of the * imports except the one in __init__.py since I was not sure about that.

Reply:
Ok I removed all of the * imports and it seems to be working alright on dev.